dvc/commands/ls/__init__.py

```diff
@@ -20,7 +22,17 @@ def _prettify(entries, with_color=False):
         def fmt(entry):
             return entry["path"]
 
-    return [fmt(entry) for entry in entries]
+    def row(entry):
+        if with_size:
+            size = entry.get("size")
+            if size is None:
+                size = ""
+            else:
+                size = tqdm.format_sizeof(size, divisor=1024)
```

Comment:
Will need to introduce some `humanize` util at some point when we'll need again.